core/server/api/shared/headers.js

```diff
@@ -0,0 +1,50 @@
+const INVALIDATE_ALL = '/*';
+
+const cacheInvalidate = (result, options = {}) => {
+    let value = options.value;
+
+    return {
+        'X-Cache-Invalidate': value || INVALIDATE_ALL
+    };
+};
+
+const disposition = {
+    csv(result, options = {}) {
+        return {
+            'Content-Disposition': options.value,
+            'Content-Type': 'text/csv'
```

Comment:
Why is `Content-Length` header omitted in this case? Both json&yaml responses return this header.

Reply:
It's a copy paste from v0.1. I have not refactored/reconsidered every bit.